rustfmt.toml

```diff
@@ -1,3 +1 @@
-group_imports = "StdExternalCrate"
-imports_granularity = "Module"
```

Comment:
I think these should be added back once .github/workflows/check.yml goes back to using nightly. Same for the other rustfmt.toml

Reply:
If they're nightly features, I suggest we don't keep them. There's recent activity on the PRs tracking i.e. the imports_granularity feature, and our code formatters changing a rule out from under us will cause future devex and CI woes.